src/components/MarkdownProvider/components/BestPractice.tsx

```diff
@@ -8,7 +8,7 @@
 import React, { ReactNode } from 'react';
 import styled from 'styled-components';
 import { math } from 'polished';
-import Img, { FluidObject } from 'gatsby-image';
```

Comment:
Can `gatsby-image` be removed from package.json?

Reply:
Yeah, it was removed 👍🏽 .